doc/server_configurations.md

```diff
@@ -7695,14 +7695,50 @@ Volar can be installed via npm
 npm install -g @volar/server
 ```
 
-With Vue 3 projects - it works out of the box.
+Vue 3 works by default, but Vue 2 needs [additional configuration](https://github.com/johnsoncodehk/volar/blob/master/extensions/vscode-vue-language-features/README.md?plain=1#L28-L63).
 
-With Vue 2 projects - requires [additional configuration](https://github.com/johnsoncodehk/volar#using)
+`volar` succeeds `vuels` (Vetur), so certain language features overlap between the two. To avoid getting duplicate completions, code actions etc. from two servers at once make sure `vuels` is not running alongside `volar`
```

Comment:
volar is the successor to? it's by the same author?

Reply:
Changed to

`volar` succeeds `vuels` (Vetur) as the main language server in the Vue ecosystem, so certain language features overlap between the two. To avoid getting duplicate completions, code actions etc. from two servers at once make sure `vuels` is not running alongside `volar`